src/styles/blocks/card.scss

```diff
@@ -0,0 +1,106 @@
+:root {
+  --main-accent: #060b35;
+  --secondary: #616070;
+  --blue-accent: #00acdc;
+  --white: #fff;
+  --border-card-color: #f3f3f3;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+}
+
+.card__tittle,
+.card__code,
+.card__reviews {
+  margin: 0;
+}
+
+.card {
+  box-sizing: border-box;
```

Comment:
You already set box-sizing for the whole document, so setting box-sizing to the card is unnecessary